src/core/reopt.jl

```diff
@@ -584,7 +584,7 @@ function add_variables!(m::JuMP.AbstractModel, p::REoptInputs)
 		dvGridPurchase[p.time_steps, 1:p.s.electric_tariff.n_energy_tiers] >= 0  # Power from grid dispatched to meet electrical load [kW]
 		dvRatedProduction[p.techs.all, p.time_steps] >= 0  # Rated production of technology t [kW]
 		dvCurtail[p.techs.all, p.time_steps] >= 0  # [kW]
-		dvProductionToStorage[p.s.storage.types.all, p.techs.all, p.time_steps] >= 0  # Power from technology t used to charge storage system b [kW]
+		dvProductionToStorage[p.s.storage.types.all, union(p.techs.ghp,p.techs.all), p.time_steps] >= 0  # Power from technology t used to charge storage system b [kW]
```

Comment:
Since gph is not in techs.all, are there other places this causes an issue? I'm looking through the code and wondering about: 

- The code nested under reopt.jl, `if !isempty(p.techs.all)` line 269. E.g. `add_ghp_constraints(m, p)` is nested under this. So does this mean GHP has to be evaluated with another tech? 
- And then just confirming that `dvSize`, `dvPurchaseSize`, `dvRatedProduction`, and `dvCurtail` don't need to also be indexed on GHP?

Reply:
Right now, I believe the answer is yes - a GHP-only scenario isn't likely to produce the result we want.  I raised issue https://github.com/NREL/REopt.jl/issues/393 to address this.

GHP does not produce electricity and the capital cost is attached to binary variables elsewhere, and GHP does not dispatch according to system sizing, so for now the four variables mentioned above do not need to be indexed on GHP.